upstream/CVE-2022-3162.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2022-3162",
+  "modified": "2022-11-08T21:33:07Z",
+  "published": "2022-11-08T21:33:07Z",
+  "summary": "Unauthorized read of Custom Resources",
+  "details": "Users authorized to list or watch one type of namespaced custom resource cluster-wide can read custom resources of a different type in the same API group without authorization. Clusters are impacted by this vulnerability if all of the following are true: 1. There are 2+ CustomResourceDefinitions sharing the same API group 2. Users have cluster-wide list or watch authorization on one of those custom resources. 3. The same users are not authorized to read another custom resource in the same API group.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.25.0"
+            },
+            {
+              "last_affected": "1.25.3"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.24.0"
+            },
+            {
+              "last_affected": "1.24.7"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.23.0"
+            },
+            {
+              "last_affected": "1.23.13"
```

Comment:
```suggestion
              "fixed": "1.23.14"
```

```bash
➜  kubernetes git:(master) git tag --contains 845565ba773c10be6a020188437bcd86936fc546 | sort -V | head -n 5
v1.23.14
v1.23.15
v1.23.15-rc.0
v1.23.16
v1.23.16-rc.0
```

Reply:
fixed in https://github.com/aquasecurity/vuln-list-k8s/pull/89/commits/005e4b871d9ccfff18cab47d1019f2756fa9546e